File: 1863.cpp
class Solution {
public:
    int subsetXORSum(vector<int>& nums) {
        int ans = 0;
        int n = size(nums);
        for(int i=0;i<(1<<n);i++){
            int c=0;
            for(int k=0;k<n;k++){
                if (i & 1<<k){c^=nums[k];}
            }
            ans+=c;
        }
        return ans;
    }
};

File: 1863.py
class Solution(object):
    def subsetXORSum(self, nums):
        ans=0
        n=len(nums)
        for i in range(1<<n):
            t=0
            for k in range(n):
                if i & 1<<k:
                    t^=nums[k]
            ans+=t
        return ans


        